repotracker/repotracker.go

```diff
@@ -580,7 +580,7 @@ func makeBuildBreakSubscriber(userID string) (*event.Subscriber, error) {
 		if preference == user.PreferenceEmail {
 			subscriber.Target = u.Email()
 		} else if preference == user.PreferenceSlack {
-			slackTarget := u.Settings.SlackUsername
+			slackTarget := fmt.Sprintf("@%s", u.Settings.SlackUsername)
```

Comment:
I would check if it already starts with an `@` to avoid double-adding it.

Reply:
true, fixed.